.travis.yml

```diff
@@ -119,30 +119,38 @@ matrix:
         - ./.travis/install-cython-examples.sh
       script:
         - export PATH="$HOME/miniconda/bin:$PATH"
+        # The following is needed so cloudpickle can find some of the
+        # class definitions: The main module of tests that are run
+        # with pytest have the same name as the test file -- and this
+        # module is only found if the test directory is in the PYTHONPATH.
+        - export PYTHONPATH="$PYTHONPATH:./test/"
```

Comment:
This line should be reverted once we convert all of the tests to pytest, right?

Reply:
Actually no, it is required for the pytest tests to run unfortunately (because cloudpickle needs to find the module associated to the test to be able to import class definitions from that file).

I'm a little bit surprised that this didn't come up for any of the other pytest tests that have been written in the past (like for RLLib for example). It only seems to happen in corner cases.